lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
@jcoyne The problem this line is trying to solve is that we need the Legato library available in three different files: analytics.rb, pageview.rb, and download.rb. These lines make sure this line is available to those three files: https://github.com/projecthydra/sufia/blob/d515cd314a2d5e9e8179302e69824299e83e43b2/app/services/sufia/analytics.rb#L3

I'm sure @njaffer is open to others ways to handle this.